packages/main/src/Table.js

```diff
@@ -191,6 +196,18 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines the mode of the component (None, SingleSelect, MultiSelect).
```

Comment:
We will revisit all the API of all the components but this can be improved on the spot, let's use the bullets representation of enum options
 
 
 * Defines the mode of the <code>ui5-table</code>.
 * <br><br>
		 * Available options are:
		 * <ul>
		 * <li><code>MultiSelect</code></li>
		 * <li><code>SingleSelect</code></li>
		 * <li><code>None</code></li>
		 * <ul>